.rubocop.yml

```diff
@@ -0,0 +1,33 @@
+inherit_from: .rubocop_todo.yml
+
+require:
+  - rubocop-performance
+  - rubocop-rails
+  - rubocop-minitest
+
+AllCops:
+  TargetRubyVersion: 2.7
+
+Layout/DotPosition:
```

Comment:
I think we should not disable or alter the rules, but rather to fix them. What do you think?